src/query/ValueExprPredicate.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of qserv.
+ *
+ * Developed for the LSST Data Management System.
+ * This product includes software developed by the LSST Project
+ * (https://www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef LSST_QSERV_VALUE_EXPR_PREDICATE_H
+#define LSST_QSERV_VALUE_EXPR_PREDICATE_H
+
+// System headers
+#include <memory>
+#include <string>
+
+// Local headers
+#include "query/Predicate.h"
+
+namespace lsst::qserv::query {
+class ValueExpr;
```

Comment:
Very minor comment. Usually, this needs to be indented by 4 spaces. Was the indentation removed by `clang-format`?